packages/fast-color-explorer/README.md

```diff
@@ -0,0 +1,16 @@
+# Introduction 
+Application for UI testing and development.
+
+# Getting Started
+Clone the repository, and then run `npm i` in the root directory.
```

Comment:
do you want to say to run `lerna bootstrap` then cd into the packages/fast-color-explorer?